pkg/handler/utils.go

```diff
@@ -24,3 +25,8 @@ func removeEndSuffix(source string, suffix string) string {
 
 	return output
 }
+
+func addRoute(e *echo.Group, method string, path string, h echo.HandlerFunc, verb rbac.Verb, m ...echo.MiddlewareFunc) {
+	e.Add(method, path, h, m...)
+	rbac.ServicePermissions.Add(method, path, rbac.ResourceRepositories, verb)
```

Comment:
note that this assumes a resource of 'Repositories'.  we can refactor if we ever have a 2nd resource type.